package/src/components/ConfirmDialog/ConfirmDialog.test.js

```diff
@@ -1,20 +1,27 @@
 import React from "react";
-import renderer from "react-test-renderer";
+import { render, fireEvent, cleanup } from "../../tests/";
+import "@testing-library/jest-dom/extend-expect";
 import Button from "../Button";
 import ConfirmDialog from "./ConfirmDialog";
 
-test("basic snapshot", () => {
-  const component = renderer.create((
+afterEach(cleanup);
+
+test("basic snapshot - with opening the dialog", () => {
```

Comment:
I'm not sure how this components works if it gets added to the DOM or if it's just hidden on the page. If it's getting attached to the DOM on open I think you could use [toBeInTheDocument](https://github.com/testing-library/jest-dom#tobeinthedocument) assertion in this test.